packages/medusa/src/api/routes/store/carts/set-payment-session.ts

```diff
@@ -13,6 +13,13 @@ import { validator } from "../../../../utils/validator"
  * parameters:
  *   - (path) id=* {string} The ID of the Cart.
  *   - (body) provider_id=* {string} The ID of the Payment Provider.
```

Comment:
This should be removed

Reply:
Good catch! Fixed in afe7924988f9908c8da1fbdfbff808002b4cb48e